pubsub/sample.rb

```diff
@@ -106,3 +106,89 @@ def list_subscriptions
   end
 end
 # [END list_subscriptions]
+
+# [START print_topic_policy]
```

Comment:
All the existing tags are around the methods https://cloud.google.com/pubsub/publisher

Do you want me to change all of them?